app/gosns/gosns.go

```diff
@@ -485,6 +491,132 @@ func DeleteTopic(w http.ResponseWriter, req *http.Request) {
 
 }
 
+func PublishBatch(w http.ResponseWriter, req *http.Request) {
```

Comment:
This looks great - would you mind also updating the Readme on it?  Like adding a box and checking it [HERE](https://github.com/Admiral-Piett/goaws/blob/master/.github/README.md?plain=1#L43-L53), and updating the Postman collection with an example request [HERE](https://github.com/Admiral-Piett/goaws/blob/master/.github/README.md?plain=1#L103)?

Reply:
How can I update the Postman collection?

Reply:
Are you using Postman to interact with goaws at all?  If you're not, I can update it for you, if you can give me a copy of your request with `curl` for reference.  It's located here - https://www.getpostman.com/collections/091386eae8c70588348e. Postman can import it if you just give it the link.

If you are, you can just include your new request in that collection and right click > select "Share" > Select "Via API" then replace the link here https://github.com/Admiral-Piett/goaws/blob/master/.github/README.md?plain=1#L113C65-L113C124 with your new link.

Happy to do it for you though if you prefer.

Reply:
In fact - it might be better if you just send me the curl?  I'm going to have to a big update there looks like.  😮‍💨